atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -436,6 +436,10 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
             if data.get(IGNORE_ABSENT_REPOS):
                 self.flags.append(IGNORE_ABSENT_REPOS)
 
+            dev_content_sets = data.get('dev_content_sets', {})
```

Comment:
Why `dev_content_set`? Shouldn't it be called `build_content_sets`?

Reply:
How is case when `dev_content_sets` is `null` handled? (Allowed by schema)

Reply:
@lcarva @athos-ribeiro @ben-alkov  ?

Reply:
added handling None